src/server.c

```diff
@@ -3562,6 +3562,11 @@ int incrCommandStatsOnError(struct redisCommand *cmd, int flags) {
     return res;
 }
 
+/* Returns true if the command is not internal, or the connection is internal. */
+static bool commandAccessibleForClient(client *c, struct redisCommand *cmd) {
```

Comment:
maybe "Visible" or "Exposed" instead of "Accessible"?
we're using this in COMMAND and MONITOR, right?